source/slow-down.ts

```diff
@@ -152,8 +147,6 @@ export const slowDown = (
 		// These settings cannot be overridden.
 		limit: 0, // We want the handler to run on every request.
 		// Disable the headers, we don't want to send them.
-		legacyHeaders: false,
-		standardHeaders: false,
```

Comment:
Instead of removing this code entirely, we should move this to above the `...notUndefinedOptions,` line.

That way it keeps the same default, but be overrideable.